hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/AbstractYarnScheduler.java

```diff
@@ -506,6 +506,7 @@ public void setEntitlement(String queue, QueueEntitlement entitlement)
   private void killOrphanContainerOnNode(RMNode node,
       NMContainerStatus container) {
     if (!container.getContainerState().equals(ContainerState.COMPLETE)) {
+      LOG.warn("Killing container " + container + " for unknown application");
```

Comment:
Will this change result in more log output? Does this mean that all apps that enter this judgment must be printed?

Reply:
Thank you very much for your contribution, but I think different people may have different understandings of the log output. This does not seem to be a general change, and I have not seen much value from this change.

Reply:
> Will this change result in more log output? Does this mean that all apps that enter this judgment must be printed?

@slfan1989  Not really, Only the orphan container which status is unfinished will be killed and logged here.

Reply:
I had a quick look at it yesterday, I'll be going through the code today.